### ISSUE TITLE
RX - Changes to List and Sort - Product - Update App Store Images

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#7391]
**Description**
As the Veteran, I want all the app stores to accurately reflect the most recent changes to the app so that what I learn in app stores is accurately reflected in the app. 

**Info**

- Determine what images need to be updated
- Determine if App Store content needs to be updated - then create a separate ticket for Content to update 
- New images need to be run through something like ImageOptim per Tim before being added into the repo to remove any metadata that might be bloating the files

**Ask:**

- [x] - Create image for [Apple Store](https://apps.apple.com/us/app/va-health-and-benefits/id1559609596?platform=iphone) for iPad
- [x] - Create image for [Apple Store](https://apps.apple.com/us/app/va-health-and-benefits/id1559609596?platform=iphone) for iPhone
- [x] - Create image for [Google Play Store](https://play.google.com/store/apps/details?id=gov.va.mobileapp&hl=en_US&gl=US)
- [x] - Create image for [VA.gov app page ](https://mobile.va.gov/app/va-health-and-benefits)\

**Comment by brea11y** [comment]
@bischoffa and/or @ala-yna – Do we want to update the content within the app store image screens? I have swapped out the old Rx screen for the new screen (list view - _chosen by Lauren_), but before I export the images, I need to know if we'd like to change the content relating to Rx. 

Currently, it says "Manage your VA prescriptions" on the image with the list of various features and in the image that features Rx. Screenshots attached of these two screens (both iPhone views, only for reference, but would apply to all screens if we change the content).

<img width="450" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeU11QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4277e8022efba1a2752b8e15e14b5cccb9f35f58/iphone55-screen-1.png" alt="Use this app to:
Securely sign in with biometrics
Manage your VA prescriptions
Manage your VA prescriptions
Review and cancel appointments
Send and receive secure messages
Download letters
Check claims and appeals status
Get your VA vaccine records
Review your payments history
Show your Veteran status" />

<img width="450" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVF1QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f8d198641a47eda84516a719a423278e5a644baa/iphone55-screen-10.png" alt="Manage your VA prescriptions" />

**Comment by bischoffa** [comment]
@brea11y when discussing this with Meko we were just intending an RX image change and didn't think a header or use this app image needed changed. I think you are good with just updating the RX image.  But will leave that up to @ala-yna for final decision.

**Comment by mistymg** [comment]
No words need changed on these screenshots. Just images. My 2 cents :)

**Comment by brea11y** [comment]
I agree with both of you. I'll leave it as-is now and can always change the copy and re-export if needed. 

Also, @bischoffa – That last ask / checkbox is not something that I am able to do. This would be up to Misty and Alayna, I believe.

**Comment by bischoffa** [comment]
Thanks @brea11y. You are correct that would be determined by the Health Team. 

Initial thought is that Va.Gov app content likely wont change since its an existing feature vs adding a new one. 

But I am assuming the project will require Whats New content in the App store based on past usage. Though its outside of this ticket will need to consider using the Whats New In-App Alert Box for content as well.

**Comment by mistymg** [comment]
Exactly right, Adam! No content changes are needed for VA.gov app store. But I have a ticket for content for the What's New section of the Google/Apple store as well as the What's New alert box in the app.

**Comment by mistymg** [comment]
@ala-yna I brought in [ticket 7712](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/7712). In that ticket is the copy for the Google/Apple What's New sections AND the in-app What's New alert box.

**Comment by brea11y** [comment]
@bischoffa and @ala-yna – The updated images are [in the Google Drive](https://drive.google.com/drive/folders/1LNcxmqrhJKcGaq8_MUHplNJ-PgArYdxN?usp=drive_link) and ready for review. I'm new to this process so I didn't know if we would need all of them or just the one that as being updated, so I exported them all. However, only the Rx images have changed.  
  
The file names that changed are: 

*   Android/8\_en-US.png
*   Apple/iPad/ipadPro129-screen-10.png
*   Apple/iPhone 6.7/iphone67-screen-10.png
*   Apple/iPhone 5.5/iphone55-screen-10.png

Please let me know if any changes are needed. If not, the images should be ready to publish.

**Comment by bischoffa** [comment]
Thanks! Team PM / Alayna will review it to approve of the images. 

@dumathane for app store image updates do we need all of them uploaded even the ones that didn't change?

**Comment by brea11y** [comment]
Perfect! The files are available in the [Google Drive](https://drive.google.com/drive/folders/1LNcxmqrhJKcGaq8_MUHplNJ-PgArYdxN?usp=drive_link). I will get this closed out. Please let me know if anything else is needed on my end!

**Comment by bischoffa** [comment]
@

**Comment by bischoffa** [comment]
@brea11y we are going to hold off on making app store changes for RX for this release. Right now QA does not have a test user to validate the work so we are playing it safe and not pushing it out.

**Comment by bischoffa** [comment]
Reopening as it seems the images never got out. Jen reached out when she saw RX images are outdated via [Slack ](https://adhoc.slack.com/archives/C02N65QRRB7/p1717423067379279)

**Comment by bischoffa** [comment]
<img width="1139" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOGN3QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1d805978034106958a505f98ebd2a5a5781786b1/image.png" alt="image.png" />

**Comment by bischoffa** [comment]
Alayna reconfirmed approval of images in the Google Folder. 

Waiting on FE to compress the images and see if it will get into this release. If so[ VA.gov App Store Page Contacts ](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/va-mobile-app/releases/Va.Gov%20APp%20Store)will be emailed with Alayna CCed asking to update images to coordinate with the release.
